Update controller cover installation to use correct length bolts.

ajacocks opened the pull request:
Update sections 00 01 for grammar, and add fastener tightening diagram.

ajacocks committed:
Update controller cover installation to use correct length bolts.
Update sections 00 01 for grammar, and add fastener tightening diagram.

adammhaile commented:
@ajacocks Nice! Very good catches there! Looks like a good old copy/paste error on the bolt lengths. Not sure how I never saw that in all my read throughs.
Many thanks!

adammhaile commented:
Wait.... @ajacocks why did you think the bolt lengths for the controller cover were wrong? Was building another one tonight and realized that 20mm was, in fact, correct. There's no way 30mm would fix there. Yes, the holes are very deep, but it's for 20mm. and you need those four 30mm bolts for the fan.

ajacocks commented:
Adam,

The longer bolts interfered with the fitting of the cover, IIRC.

Unfortunately,  my memory is foggy on the rest.

- Alex

On Sat, Jan 18, 2020, 11:09 PM Adam Haile <notifications@github.com> wrote:

> Wait.... @ajacocks <https://github.com/ajacocks> why did you think the
> bolt lengths for the controller cover were wrong? Was building another one
> tonight and realized that 20mm was, in fact, correct. There's no way 30mm
> would fix there. Yes, the holes are very deep, but it's for 20mm. and you
> need those four 30mm bolts for the fan.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4VMWRFCN5CQHWFNHVTQ6PG7VA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKIQHI#issuecomment-575965213>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFLVO4TB57EADI3S3S6MADTQ6PG7VANCNFSM4J3A7DQQ>
> .
>

adammhaile commented:
Well, what you changed it to was to the longer bolts. I've changed it back
to 20mm.
No worries. Just curious.

On Wed, Jan 22, 2020 at 11:08 AM J. Alexander Jacocks <
notifications@github.com> wrote:

> Adam,
>
> The longer bolts interfered with the fitting of the cover, IIRC.
>
> Unfortunately, my memory is foggy on the rest.
>
> - Alex
>
> On Sat, Jan 18, 2020, 11:09 PM Adam Haile <notifications@github.com>
> wrote:
>
> > Wait.... @ajacocks <https://github.com/ajacocks> why did you think the
> > bolt lengths for the controller cover were wrong? Was building another
> one
> > tonight and realized that 20mm was, in fact, correct. There's no way 30mm
> > would fix there. Yes, the holes are very deep, but it's for 20mm. and you
> > need those four 30mm bolts for the fan.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4VMWRFCN5CQHWFNHVTQ6PG7VA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKIQHI#issuecomment-575965213
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AFLVO4TB57EADI3S3S6MADTQ6PG7VANCNFSM4J3A7DQQ
> >
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2X4SPSGDZHIKTXTFODQ7BVORA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUEQDI#issuecomment-577259533>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHJV2TP2USX5P7MEFMTK53Q7BVORANCNFSM4J3A7DQQ>
> .
>

ajacocks commented:
With the 20s, on mine, the threads wouldn't engage.

That's why I changed it.

I dunno what the difference would be, though.

- Alex

On Wed, Jan 22, 2020 at 1:42 PM Adam Haile <notifications@github.com> wrote:

> Well, what you changed it to was to the longer bolts. I've changed it back
> to 20mm.
> No worries. Just curious.
>
> On Wed, Jan 22, 2020 at 11:08 AM J. Alexander Jacocks <
> notifications@github.com> wrote:
>
> > Adam,
> >
> > The longer bolts interfered with the fitting of the cover, IIRC.
> >
> > Unfortunately, my memory is foggy on the rest.
> >
> > - Alex
> >
> > On Sat, Jan 18, 2020, 11:09 PM Adam Haile <notifications@github.com>
> > wrote:
> >
> > > Wait.... @ajacocks <https://github.com/ajacocks> why did you think the
> > > bolt lengths for the controller cover were wrong? Was building another
> > one
> > > tonight and realized that 20mm was, in fact, correct. There's no way
> 30mm
> > > would fix there. Yes, the holes are very deep, but it's for 20mm. and
> you
> > > need those four 30mm bolts for the fan.
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4VMWRFCN5CQHWFNHVTQ6PG7VA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKIQHI#issuecomment-575965213
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AFLVO4TB57EADI3S3S6MADTQ6PG7VANCNFSM4J3A7DQQ
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2X4SPSGDZHIKTXTFODQ7BVORA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUEQDI#issuecomment-577259533
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAHJV2TP2USX5P7MEFMTK53Q7BVORANCNFSM4J3A7DQQ
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4WBURQDALFJGA7DGY3Q7CHRFA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUVBNA#issuecomment-577327284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFLVO4VADXVXAHIPKWSHDO3Q7CHRFANCNFSM4J3A7DQQ>
> .
>


-- 
======================================================================
Alexander Jacocks, RHCE           | Staff Solution Architect
(e) alexander@redhat.com          | Public Sector Business Development
(c) (240) 447-5974                | Tyson's Corner, VA
(g) CC28 8136 C253 907A B9BB  9EEA D3C8 1A3A 790A 0AB9

adammhaile commented:
.... Remind me, did you print your own parts?
The 30s should be too long by a lot. I checked the original design and it's
definitely 20mm
Plus if you used the 30s for the controller cover you wouldn't have the
right length for the fan.

On Wed, Jan 22, 2020 at 4:11 PM J. Alexander Jacocks <
notifications@github.com> wrote:

> With the 20s, on mine, the threads wouldn't engage.
>
> That's why I changed it.
>
> I dunno what the difference would be, though.
>
> - Alex
>
> On Wed, Jan 22, 2020 at 1:42 PM Adam Haile <notifications@github.com>
> wrote:
>
> > Well, what you changed it to was to the longer bolts. I've changed it
> back
> > to 20mm.
> > No worries. Just curious.
> >
> > On Wed, Jan 22, 2020 at 11:08 AM J. Alexander Jacocks <
> > notifications@github.com> wrote:
> >
> > > Adam,
> > >
> > > The longer bolts interfered with the fitting of the cover, IIRC.
> > >
> > > Unfortunately, my memory is foggy on the rest.
> > >
> > > - Alex
> > >
> > > On Sat, Jan 18, 2020, 11:09 PM Adam Haile <notifications@github.com>
> > > wrote:
> > >
> > > > Wait.... @ajacocks <https://github.com/ajacocks> why did you think
> the
> > > > bolt lengths for the controller cover were wrong? Was building
> another
> > > one
> > > > tonight and realized that 20mm was, in fact, correct. There's no way
> > 30mm
> > > > would fix there. Yes, the holes are very deep, but it's for 20mm. and
> > you
> > > > need those four 30mm bolts for the fan.
> > > >
> > > > —
> > > > You are receiving this because you were mentioned.
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4VMWRFCN5CQHWFNHVTQ6PG7VA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKIQHI#issuecomment-575965213
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AFLVO4TB57EADI3S3S6MADTQ6PG7VANCNFSM4J3A7DQQ
> > > >
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2X4SPSGDZHIKTXTFODQ7BVORA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUEQDI#issuecomment-577259533
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAHJV2TP2USX5P7MEFMTK53Q7BVORANCNFSM4J3A7DQQ
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4WBURQDALFJGA7DGY3Q7CHRFA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUVBNA#issuecomment-577327284
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AFLVO4VADXVXAHIPKWSHDO3Q7CHRFANCNFSM4J3A7DQQ
> >
> > .
> >
>
>
> --
> ======================================================================
> Alexander Jacocks, RHCE | Staff Solution Architect
> (e) alexander@redhat.com | Public Sector Business Development
> (c) (240) 447-5974 | Tyson's Corner, VA
> (g) CC28 8136 C253 907A B9BB 9EEA D3C8 1A3A 790A 0AB9
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2S7IGPDBROEVJ2TDEDQ7CY7DA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJVDWPQ#issuecomment-577387326>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHJV2WFXP6YIG5BOJPWCX3Q7CY7DANCNFSM4J3A7DQQ>
> .
>

ajacocks commented:
I printed my own. Memories are definitely hazy, but I’ll pull one of the
bolts out and confirm.

- Alex

On Wed, Jan 22, 2020 at 16:14 Adam Haile <notifications@github.com> wrote:

> .... Remind me, did you print your own parts?
> The 30s should be too long by a lot. I checked the original design and it's
> definitely 20mm
> Plus if you used the 30s for the controller cover you wouldn't have the
> right length for the fan.
>
> On Wed, Jan 22, 2020 at 4:11 PM J. Alexander Jacocks <
> notifications@github.com> wrote:
>
> > With the 20s, on mine, the threads wouldn't engage.
> >
> > That's why I changed it.
> >
> > I dunno what the difference would be, though.
> >
> > - Alex
> >
> > On Wed, Jan 22, 2020 at 1:42 PM Adam Haile <notifications@github.com>
> > wrote:
> >
> > > Well, what you changed it to was to the longer bolts. I've changed it
> > back
> > > to 20mm.
> > > No worries. Just curious.
> > >
> > > On Wed, Jan 22, 2020 at 11:08 AM J. Alexander Jacocks <
> > > notifications@github.com> wrote:
> > >
> > > > Adam,
> > > >
> > > > The longer bolts interfered with the fitting of the cover, IIRC.
> > > >
> > > > Unfortunately, my memory is foggy on the rest.
> > > >
> > > > - Alex
> > > >
> > > > On Sat, Jan 18, 2020, 11:09 PM Adam Haile <notifications@github.com>
> > > > wrote:
> > > >
> > > > > Wait.... @ajacocks <https://github.com/ajacocks> why did you think
> > the
> > > > > bolt lengths for the controller cover were wrong? Was building
> > another
> > > > one
> > > > > tonight and realized that 20mm was, in fact, correct. There's no
> way
> > > 30mm
> > > > > would fix there. Yes, the holes are very deep, but it's for 20mm.
> and
> > > you
> > > > > need those four 30mm bolts for the fan.
> > > > >
> > > > > —
> > > > > You are receiving this because you were mentioned.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <
> > > >
> > >
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4VMWRFCN5CQHWFNHVTQ6PG7VA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKIQHI#issuecomment-575965213
> > > > >,
> > > > > or unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AFLVO4TB57EADI3S3S6MADTQ6PG7VANCNFSM4J3A7DQQ
> > > > >
> > > > > .
> > > > >
> > > >
> > > > —
> > > > You are receiving this because you modified the open/close state.
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2X4SPSGDZHIKTXTFODQ7BVORA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUEQDI#issuecomment-577259533
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AAHJV2TP2USX5P7MEFMTK53Q7BVORANCNFSM4J3A7DQQ
> > > >
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4WBURQDALFJGA7DGY3Q7CHRFA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJUVBNA#issuecomment-577327284
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AFLVO4VADXVXAHIPKWSHDO3Q7CHRFANCNFSM4J3A7DQQ
> > >
> > > .
> > >
> >
> >
> > --
> > ======================================================================
> > Alexander Jacocks, RHCE | Staff Solution Architect
> > (e) alexander@redhat.com | Public Sector Business Development
> > (c) (240) 447-5974 | Tyson's Corner, VA
> > (g) CC28 8136 C253 907A B9BB 9EEA D3C8 1A3A 790A 0AB9
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AAHJV2S7IGPDBROEVJ2TDEDQ7CY7DA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJVDWPQ#issuecomment-577387326
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAHJV2WFXP6YIG5BOJPWCX3Q7CY7DANCNFSM4J3A7DQQ
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/Engravinator.com/pull/44?email_source=notifications&email_token=AFLVO4TSOBNTFJJVSHDD2ILQ7CZKDA5CNFSM4J3A7DQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJVD7SQ#issuecomment-577388490>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFLVO4VQAE5RENH3UXV44PLQ7CZKDANCNFSM4J3A7DQQ>
> .
>
-- 
======================================================================
Alexander Jacocks, RHCE           | Staff Solution Architect
(e) alexander@redhat.com          | Public Sector Business Development
(c) (240) 447-5974                | Tyson's Corner, VA
(g) CC28 8136 C253 907A B9BB  9EEA D3C8 1A3A 790A 0AB9
